app/src/forms/form/exportService.js

```diff
@@ -138,13 +138,18 @@ const service = {
     return `${form.snake()}_${type}.${format}`.toLowerCase();
   },
 
-  _submissionsColumns: (form) => {
+  _submissionsColumns: (form, params) => {
     // Custom columns not defined - return default column selection behavior
     let columns = ['confirmationId', 'formName', 'version', 'createdAt', 'fullName', 'username', 'email'];
     // if form has 'status updates' enabled in the form settings include these in export
     if (form.enableStatusUpdates) {
       columns = columns.concat(['status', 'assignee', 'assigneeEmail']);
     }
+    // Let's add form level columns like deleted or draft
+    if (params?.columns?.length) {
+      let optionalAcceptedColumns = ['draft', 'deleted', 'updatedAt']; //'draft', 'deleted', 'updatedAt' columns needed for ETL process at this moment
+      columns = columns.concat([...params.columns].filter((column) => optionalAcceptedColumns.includes(column)));
+    }
```

Comment:
It would be good to have tests that cover this new code. When you run the tests the coverage report for the API is generated and put into /app/coverage/lcov-report/index.html